docs/user/quickstart.rst

```diff
@@ -150,9 +150,7 @@ There's also a builtin JSON decoder, in case you're dealing with JSON data::
 
 In case the JSON decoding fails, ``r.json()`` raises an exception. For example, if
 the response gets a 204 (No Content), or if the response contains invalid JSON,
-attempting ``r.json()`` raises ``simplejson.JSONDecodeError`` if simplejson is
-installed or raises ``ValueError: No JSON object could be decoded`` on Python 2 or
-``json.JSONDecodeError`` on Python 3.
+attempting ``r.json()`` raises ``requests.JSONDecodeError``.
```

Comment:
We must be clear that this is _new_ in the next version of Requests (add it to our changelog too) and that it is backwards compatible as it inherits from the exceptions raised previously

Reply:
Ok, I'll indicate that.

Reply:
Do you want me to add it to the changelog now or do you like to do that?

Reply:
Please add it to the changelog now

Reply:
Ok, will do!

Reply:
Include it under the `dev` category?